File: VideoIntelligence-UI/admin/src/types/user/sysUserDTO.ts
/**
 * 用户新增/修改入参对象（可传入密码）
 */
export interface SysUserDTO {
  /** 用户ID（修改时必填） */
  userId?: number

  /** 用户账号 */
  userName?: string

  /** 用户昵称 */
  nickName?: string

  /** 用户邮箱 */
  email?: string

  /** 头像地址 */
  avatar?: string

  /** 密码（新增时必填，修改时选填） */
  password?: string

  /** 账号状态（0正常 1停用） */
  status?: string

  /** 备注 */
  remark?: string

  /** 角色列表 */
  roleIds?: number[]
}

File: dotm-auth/dotm-auth-client/src/main/java/com/dotm/entity/dto/SysUserDTO.java
package com.dotm.entity.dto;

import lombok.Data;

import java.util.List;

/**
 * @author dotm
 * 用户新增/修改入参对象（可传入密码）
 */
@Data
public class SysUserDTO {
    /**
     * 用户ID（修改时必填）
     */
    private Long userId;

    /**
     * 用户账号
     */
    private String userName;

    /**
     * 用户昵称
     */
    private String nickName;

    /**
     * 用户邮箱
     */
    private String email;

    /**
     * 头像地址
     */
    private String avatar;

    /**
     * 密码（新增时必填，修改时选填）
     */
    private String password;

    /**
     * 账号状态（0正常 1停用）
     */
    private String status;

    /**
     * 备注
     */
    private String remark;

    /**
     * 用户角色ID集合
     */
    private List<Long> roleIds;
}